server/coordinator.go

```diff
@@ -115,15 +114,16 @@ func (c *coordinator) patrolRegions() {
 	defer logutil.LogPanic()
 
 	defer c.wg.Done()
-	ticker := time.NewTicker(patrolRegionInterval)
-	defer ticker.Stop()
+	timer := time.NewTimer(c.cluster.GetPatrolRegionInterval())
+	defer timer.Stop()
 
 	log.Info("coordinator: start patrol regions")
-
+	start := time.Now()
 	var key []byte
 	for {
 		select {
-		case <-ticker.C:
+		case <-timer.C:
+			timer.Reset(c.cluster.GetPatrolRegionInterval())
```

Comment:
`case <-time.After(c.cluster.GetPatrolRegionInterval())` seems simpler.

Reply:
What's the benefit of using `Timer` instead of `Ticker`?

Reply:
can dynamically adjust the interval via `pd-ctl`